app/load.server.ts

```diff
@@ -111,33 +114,42 @@ export const loadDataForRegion = async (
     })
     .sort((a, b) => a.ts - b.ts);
 
+  if (process.env.NODE_ENV === "development") {
```

Comment:
Shouldn't this be `!==`? Otherwise, you're not ever putting into the cache in production.
```suggestion
  if (process.env.NODE_ENV !== "development") {
```